move/axelar_gateway/sources/gateway.move

```diff
@@ -58,8 +58,6 @@ const ENotLatestSigners: u64 = 3;
 const COMMAND_TYPE_APPROVE_MESSAGES: u8 = 0;
 const COMMAND_TYPE_ROTATE_SIGNERS: u8 = 1;
```

Comment:
would be nice to convert this to enums as well. Although I don't know if sui enums expose a convenient enum -> u8 conversion or via casting. can you check or ask mysten team if they plan to add that?

Reply:
seems like the current approach increases code complexity due to the need for manual handling of u8 <-> enums conversion. i've already asked mysten team about this.